alpha/declcfg/declcfg.go

```diff
@@ -90,6 +92,19 @@ type RelatedImage struct {
 	Image string `json:"image"`
 }
 
+type Deprecation struct {
+	Schema       string             `json:"schema"`
+	Package      string             `json:"package"`
+	Name         string             `json:"name,omitempty"`
+	Deprecations []DeprecationEntry `json:"deprecations"`
+}
+
+type DeprecationEntry struct {
+	Schema  string          `json:"schema"`
+	Name    string          `json:"name,omitempty"`
+	Message json.RawMessage `json:"message"`
```

Comment:
I know this is already merged, but:
```suggestion
	Message string `json:"message"`
```